queries/ecma/injections.scm

```diff
@@ -43,6 +43,25 @@
   (#set! injection.include-children)
   (#set! injection.language "html"))
 
+; Vercel PostgreSQL
+; foo.sql`...` or foo.sql(`...`)
+(call_expression
+  function: [
+    (await_expression
+      (member_expression
+        property: (property_identifier) @injection.language))
```

Comment:
Out of curiosity, what does this capture try to achieve? Looks like without it the injection still works for something like ``await client.sql`...`;``

Reply:
Without that line the query doesn't capture async calls with type arguments (e.g., `` await client.sql<T>`...` ``).